sdk/openai/Azure.AI.OpenAI/src/Custom/Utf8TStreamReader.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Threading;
+
+namespace Azure.Core.Sse
+{
+    public sealed class Utf8TStreamReader : IDisposable
```

Comment:
You don't need this type in the preview. I just started to work on it to avoid unecessary UTF8-UTF16-UTF8 conversions, but it's not yet used by the reader.

Reply:
Thanks; should've looked more carefully during the port. Removed for now.